lib/guides_style_18f/includes/header.html

```diff
@@ -3,6 +3,51 @@
   <meta charset="utf-8">
   <title>{{ page.title }} - {{ site.name }}</title>
   <meta name="viewport" content="width=device-width">
+
+
+
+{% if site.url %}
```

Comment:
It's a way to make this backwards-compatible and opt-in. I could add a bunch of fences, but this seemed cleaner. I can do that, though.